semcore/notice/src/style/notice.shadow.css

```diff
@@ -126,3 +126,16 @@ SNotice[theme='custom'] {
   background-color: var(--backgroundColor);
   border-color: var(--backgroundColor);
 }
+
+STitle {
+  font-size: var(--intergalactic-fs-300, 16px);
+  font-weight: var(--intergalactic-bold, 700);
+  line-height: var(--intergalactic-lh-300, 150%);
```

Comment:
As you changed render components, customise them.
You don't need `font-size`, `font-weight`, `line-height`

Reply:
Not sure i understand you. These rules are needed to distinguish title style form other text.

Reply:
moved styles to props